scripts.js

```diff
@@ -19,3 +20,205 @@ img.addEventListener('click', toggleNav);
+    excerpt:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    tools: ['html', 'css', 'javascript'],
+    submitButton: 'work-section-button',
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    id: 'proj2',
+    name: 'Multi-Post Stories',
+    background: 'CANOPY',
+    stack: 'Back End Dev',
+    year: '2015',
+    image: 'images/Snapshoot Portfolio (1).svg',
+    alt: 'Snapshoot Portfolio image',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    excerpt:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    tools: ['html', 'css', 'javascript'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    id: 'proj3',
+    name: 'Tonic',
+    background: 'CANOPY',
+    stack: 'Back End Dev',
+    year: '2015',
+    image: 'images/Snapshoot Portfolio (2).svg',
+    alt: 'Snapshoot Portfolio image',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    excerpt:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    tools: ['html', 'css', 'javascript'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    id: 'proj4',
+    name: 'Multi-Post Stories',
+    background: 'CANOPY',
+    stack: 'Back End Dev',
+    year: '2015',
+    image: 'images/Snapshoot Portfolio (3).svg',
+    alt: 'Snapshoot Portfolio image',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    excerpt:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    tools: ['html', 'css', 'javascript'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+];
+// Tools template
+function getToolsTemplate(tools) {
```

Comment:
- [ ] I would also recommend you to use arrow function instead of the function keyword. The arrow function is ES6 which shows you are updated on the latest version usage of JS. You can read on it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)

Reply:
Okay. Thank you.